Core/Utilities/MySQL.py

```diff
@@ -458,7 +458,7 @@ def __escapeString( self, myString ):
     It also includes quotation marks " around the given string
     """
 
-    retDict = self.__getConnection()
+    retDict = self._getConnection()
```

Comment:
This method already existed, and was just calling the private method. I simplified the code.